formapp/templates/controlpanel.html

```diff
@@ -224,7 +209,7 @@ <h2>Permissions</h2>
     <div id="permsContainer">
       <div class="permBox">
         <h3>Chat</h3>
-        <select onchange="updatePerm(this);" id="saypermsDrop" name="saypermsDrop" style="margin-top: 8px;">
+        <select onchange="updatePerm(this)" id="saypermsDrop" name="saypermsDrop" style="margin-top: 8px;">
```

Comment:
Wasn't neccesary

Reply:
I dont think this was mine and I do not know what that does.